config/packages/prod/Centreon.yaml

```diff
@@ -253,10 +253,14 @@ services:
     Centreon\Domain\RemoteServer\Interfaces\RemoteServerServiceInterface:
         class: Centreon\Domain\RemoteServer\RemoteServerService
 
-    Centreon\Domain\RemoteServer\Interfaces\RemoteServerRepositoryInterface:
+    Centreon\Domain\RemoteServer\Interfaces\RemoteServerRepositoryFileInterface:
```

Comment:
The name of an interface should remain abstract and should not provide technical information. The term 'File' is a technical indication.